dockstore-webservice/src/main/resources/migrations.1.16.0.xml

```diff
@@ -191,4 +191,42 @@
         <modifyDataType columnName="topicmanual" newDataType="varchar(250)" tableName="tool"/>
         <modifyDataType columnName="topicmanual" newDataType="varchar(250)" tableName="workflow"/>
     </changeSet>
+    <changeSet author="hyunnaye" id="fixConceptDOIs">
+        <sql dbms="postgresql">
+            ALTER TABLE workflow DISABLE TRIGGER update_archived_workflow;
+
+            UPDATE workflow
+            SET conceptdoi = REGEXP_REPLACE(conceptdoi, 'doi/', '', 'g')
+            WHERE conceptdoi IS NOT NULL;
+
+            ALTER TABLE workflow ENABLE TRIGGER update_archived_workflow;
+        </sql>
+        <sql dbms="postgresql">
+            ALTER TABLE apptool DISABLE TRIGGER update_archived_apptool;
+
+            UPDATE apptool
+            SET conceptdoi = REGEXP_REPLACE(conceptdoi, 'doi/', '', 'g')
+            WHERE conceptdoi IS NOT NULL;
+
+            ALTER TABLE apptool ENABLE TRIGGER update_archived_apptool;
+        </sql>
+        <sql dbms="postgresql">
+            ALTER TABLE service DISABLE TRIGGER update_archived_service;
```

Comment:
Have these migrations been tested on qa? 
Asking because I thought migrations are run via the dockstore user 
https://github.com/dockstore/compose_setup/blob/5c59a56af0cecaea4887bd805aba7277ab7e58b8/templates/init_migration.sh.template

I don't recall if the dockstore user has the ability to disable the frozen trigger

Reply:
It looks like we've done it [in the past](https://github.com/dockstore/dockstore/blob/e152df9e4d5b81742b5729f440f5201081e6d720/dockstore-webservice/src/main/resources/migrations.1.12.0.xml#L89), as the dockstore user, so I think it's OK (but definitely test). You should be able to test locally if it's easier, if you have your postgres and dockstore DB users set up correctly.

Reply:
I tested it locally and the migrations work :)